### ISSUE TITLE
Associate form labels with form fields

### ISSUE THREAD

**Issue by chrsmrtn0** [CityOfBoston/onebostonday#15]
**URL:** http://www.onebostonday.org/community/#planning
**Found:** The labeled titles for each input field within the .planning form aren't properly associated for accessibility. This means a screen reader won't pick up what a user should be entering into these inputs. 

**Suggested:** add 'for' attributes to all labels and 'id' attributes to all inputs.

**Comment by RobErskine** [comment]
@chrsmrtn0 Up on staging as part of  commit: [1a419a6751](https://github.com/CityOfBoston/onebostonday/commit/1a419a67514d5b20974903f77c303af7de4f8080)

**Comment by chrsmrtn0** [comment]
Confirmed, form fields are labeled correctly
